docs/gettingstarted/modfiles.md

```diff
@@ -0,0 +1,160 @@
+
+[[mods]]
+  modId="examplemod"
+  version="1.0.0.0"
+  displayName="Example Mod"
+  updateJSONURL="https://files.minecraftforge.net/net/minecraftforge/forge/promotions_slim.json"
+  displayURL="https://minecraftforge.net"
+  logoFile="logo.png"
+  credits="I'd like to thank my mother and father."
+  authors="Author"
+  description='''
+  Lets you craft dirt into diamonds. This is a traditional mod that has existed for eons. It is ancient. The holy Notch created it. Jeb rainbowfied it. Dinnerbone made it upside down. Etc.
+  '''
+  displayTest="MATCH_VERSION"
+
+[[dependencies.examplemod]]
+  modId="forge"
+  mandatory=true
+  versionRange="[45,)"
+  ordering="NONE"
+  side="BOTH"
+
+[[dependencies.examplemod]]
+  modId="minecraft"
+  mandatory=true
+  versionRange="[1.19.4]"
+  ordering="NONE"
+  side="BOTH"
+```
+
+`mods.toml` is broken into three parts: the non-mod-specific properties, which are linked to the mod file; the mod properties, with a section for each mod; and the dependency configurations, with a section for each mod's or mods' dependencies. Each of the properties associated with the `mods.toml` file will be explained below, where `required` means that a value must be specified or an exception will be thrown.
+
+### Non-Mod-Specific Properties
+
+Non-mod-specific properties are properties associated with the JAR itself,  indicating how to load the mod(s) and any additional global metadata.
+
+Property             | Type    | Default       | Description | Example
+:---                 | :---:   | :---:         | :---:       | :---
+`modLoader`          | string  | **mandatory** | The language loader used by the mod(s). Can be used to support alternative language structures, such as Kotlin objects for the main file, or different methods of determining the entrypoint, such as an interface or method. Forge provides the Java loader `"javafml"` and low/no code loader `"lowcodefml"`. | `"javafml"`
+`loaderVersion`      | string  | **mandatory** | The acceptable version range of the language loader, expressed as a [Maven Version Range][mvr]. For `javafml` and `lowcodefml`, the version is the major version of the Forge version. | `"[45,)"`
+`license`            | string  | **mandatory** | The license the mod(s) in this JAR are provided under. It is suggested that this is set to the [SPDX identifier][spdx] you are using and/or a link to the license. You can visit https://choosealicense.com/ to help pick the license you want to use. | `"MIT"`
+`showAsResourcePack` | boolean | `false`       | When `true`, the mod(s)'s resources will be displayed as a separate resource pack on the 'Resource Packs' menu, rather than being combined with the 'Mod resources' pack. | `true`
+`services`           | array   | `[]`          | An array of services your mod uses. This is consumed as part of the created module for the mod from Forge's implementation of the Java Platform Module System. | `["net.minecraftforge.forgespi.language.IModLanguageProvider"]`
```

Comment:
The services are detected from the declared service files, usage of this property should be avoided.

Reply:
Why do we have the property if it should be avoided regardless?